lib/sequelize.js

```diff
@@ -1093,5 +1119,7 @@ module.exports = (function() {
     this.connectionManager.close();
   };
 
+  // Allows the promise to access cls namespaces
+  Promise.Sequelize = Sequelize;
```

Comment:
@cusspvz As mick mentioned, no all instances share the same namespace. This is clearly documented, and should never cause any problems with variable leaks since different instances will naturally use different contexts just like different transactions will